.ci/run-container-ci

```diff
@@ -25,7 +25,7 @@
 set -e
 set -x
 
-CONTAINER=shiftcrypto/firmware_v2:18
+CONTAINER=shiftcrypto/firmware_v2:19
```

Comment:
Do the same in dockerenv? https://github.com/digitalbitbox/bitbox02-firmware/blob/ddcdb0e/scripts/dockerenv.sh#L27.

There's also `docker-compose.yml` in the root of the repo and it references unversioned image. Is that compose config file even used?

Reply:
dockerenv is deliberately at unversioned, so it works after `make dockerinit` and `docker pull shiftcrypto/firmware_v2`.

I never used docker-compose.yml, so let's delete it.

Reply:
:+1: